### PR TITLE
Feat: Change New Tab button #95

### PR DESCRIPTION
**Pull request by SimYunSup**
Fix like #95

### CODE REVIEW

**Comment by black7375** [comment]
If there is https://github.com/black7375/Firefox-UI-Fix/pull/96/commits/07f581d9fff6284a772cab8c0b3f264ed423a853, vertical alignment is not correct.
image

There are a few more conditions to look like a `tab`, which will be resolved with `force-push`.

1. Gap
It's a continuous component called a tab, not a button, so the spacing can be reduced.
we should reduce this to 1px.
https://github.com/black7375/Firefox-UI-Fix/commit/8ea735862467ebc52800649b76531b8b1e0ace35#diff-2f47ee8e5f8520c45fa369ed4e435af21b8273494d3d81eaa9b72ed77ca2f635R258

2. Bottom rounded corner
When you click on a new tab, it is moved to a new tab and **selected**.
It needs to be implied that it will be a choice.
Therefore, Bottom round corner should be included like australis https://github.com/black7375/Firefox-UI-Fix/issues/95#issuecomment-861929414.

**Comment by SimYunSup** [comment]
1. behavior at 89 means not work on alpha version, but work on nightly version. So I changed with `padding`. And It must be with `!important`, but I missed it.
2. I will try to change like [#95 (comment)](https://github.com/black7375/Firefox-UI-Fix/issues/95#issuecomment-861929414). But as you know, It will take a long time.

**Comment by black7375** [comment]
> But as you know, It will take a long time.

Yes. Bottom rounded corner is some tricky.
It's okay.

File: userChrome.css [hunk 1]
```diff
@@ -338,6 +339,26 @@
...
   }
 
+  /* Coner Rounding */
+  #tabs-newtab-button:hover {
+    /* Color */
+    fill: color-mix(in srgb, currentColor 17%, transparent) !important; /* Hardcoded compatibility var(--toolbarbutton-hover-background) */
+    -moz-context-properties: fill !important;
+
+    /* Corner Image */
+    --newtab-position: calc((var(--tab-corner-rounding) / 2) * -1);
```
Comment (SimYunSup) [inline userChrome.css:353]: There should be no decimal point. This value is `2.5px`, so It is not symmetrical.
image

Comment (black7375) [reply userChrome.css:353]: I got it.

Nox, I did a comparison.
`--tab-corner-rounding: 4px`
image

`--tab-corner-rounding: 6px`
image

tab => `4px`, newtab button => `6px`
image

Personally, I like the last one.

**Comment by black7375** [comment]
Now I think we're ready to merge. Would you like to test it?
image
image
image
image

**Comment by SimYunSup** [comment]
There is a bug like below.
image
We should add `transform` or `margin`.

In addition, margin appears not to be applied, but you can actually see it applied.

**Comment by black7375** [comment]
That's weird.
On my computer it looks fine.
image

Windows isn't available yet, so I'll test it later.

---

If we use `translateY()`, then seperator(divide line) is not displayed.

**Comment by black7375** [comment]
Applied some patches for compatibility.

**Comment by SimYunSup** [comment]
It works perfect on windows.

**Comment by black7375** [comment]
I moved to the `Major Contributors` list because it helped me to present the initial ideas and build a fairly complex feature and .

**Pull request merged**